e2e/testcafe-devextreme/tests/accessibility/drawer.ts

```diff
@@ -0,0 +1,33 @@
+import { Properties } from 'devextreme/ui/drawer.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  height: [400],
+  disabled: [true, false],
+  hint: [undefined, 'hint'],
+  revealMode: ['slide', 'expand'],
+  opened: [true, false],
+  position: ['top', 'bottom', 'left', 'right'],
```

Comment:
Now this option has no effect on adding new html elements to the component. Only the position of one element is changed. This will not be able to cause an Axe error ever, I think. So let's skip this option.